calculated_policies/aws_vpc_public_routes/main.tf

```diff
@@ -0,0 +1,51 @@
+# Smart Folder Definition
+resource "turbot_smart_folder" "aws_vpc_public_route_check" {
+  title       = var.smart_folder_title
+  description = var.smart_folder_description
+  parent      = var.smart_folder_parent_resource
+}
+
+# Calculated policy to check route tables for public routes
+resource "turbot_policy_setting" "vpc_route_table_approved_usage"{
+  resource = turbot_smart_folder.aws_vpc_public_route_check.id
+  type = "tmod:@turbot/aws-vpc-core#/policy/types/routeTableApprovedUsage"
+  template_input = <<EOT
+{
+  routeTable{
+    Routes{
+      DestinationCidrBlock
+      GatewayId
+    }
+  }
+}
+  EOT
+  template      = <<EOT
+{%- set cidrMatches = false -%}
```

Comment:
@choxie 

I have a pretty way on how to structure this code. If you request a meeting with you I can train you up.